### ISSUE TITLE
[3.1.1] Undefined variable "$ratio"

### ISSUE THREAD

**Issue by Macxim** [zurb/foundation#6207]
Hi, I'm coming back to an old website built with Foundation 3.1.1.
I have the following issue:

```shell
❯ bundle install             
Fetching gem metadata from http://rubygems.org/...........
Resolving dependencies...
Installing rake 10.4.2
Installing chunky_png 1.3.3
Installing multi_json 1.10.1
Installing sass 3.4.10
Installing compass-core 1.0.3
Installing compass-import-once 1.0.5
Installing rb-fsevent 0.9.4
Installing ffi 1.9.6
Installing rb-inotify 0.9.5
Installing compass 1.0.3
Installing modular-scale 2.0.0
Installing zurb-foundation 3.1.1
Using bundler 1.7.4
Your bundle is complete!
Use `bundle show [gemname]` to see where a bundled gem is installed.
Post-install message from compass:
    Compass is charityware. If you love it, please donate on our behalf at http://umdf.org/compass Thanks!
                                                                                                                                                                                             
~/Code/***/public/assets master*  49s  ⇣⇡
❯ sh watch.sh
>>> Sass is watching for changes. Press Ctrl-C to stop.
      error /Library/Ruby/Gems/2.0.0/gems/zurb-foundation-3.1.1/scss/foundation/common/_forms.scss (Line 6: Undefined variable: "$ratio".)
```
And my Gemfile is:

```ruby
source "http://rubygems.org"
gem "zurb-foundation", "3.1.1"
gem 'rb-fsevent', '~> 0.9.1'
```
Nothing really fancy.
I found out that the issue is related to `modular-scale` not being imported or something.  

Do you have any clue?

**Comment by gakimball** [comment]
Unfortunately we don't support Foundation 3 anymore, but I'll point you in the right direction.

Foundation 3 used the `modular-scale` gem, which is a Sass library for responsive typography. The current version of the library is 2.0.0, but that came out in 2013, and Foundation 3 was written in 2012. So Foundation is referencing a newer version of the library than it was designed for.

You'll need to explicitly specify an older version of `modular-scale` in your Gemfile, most likely 0.0.4, which [seems to be](https://github.com/at-import/modular-scale/releases) the last version released before 2.0.0.

Hope that helps!

**Comment by Macxim** [comment]
Hi @gakimball, 
 
Thanks for the advice.
I made it work with the following Gemfile.  
```ruby
source "http://rubygems.org"
gem "zurb-foundation", "3.1.1"
gem 'compass', '0.12.2'
gem 'modular-scale', '1.0.2'
gem 'rake', '>= 0'
gem 'sass', '3.2.10'
gem 'rb-fsevent', '~> 0.9'
```
